plugins/catalog-backend/src/ingestion/processors/BuiltinKindsEntityProcessor.test.ts

```diff
@@ -124,12 +144,13 @@ describe('BuiltinKindsEntityProcessor', () => {
           owner: 'o',
           lifecycle: 'l',
           definition: 'd',
+          system: 's',
```

Comment:
we actually said to not do APIs in systems yet, right?

Reply:
No we haven't. This says that an API can belong to a system. But system cannot expose or consume APIs

Reply:
Aha! Yeah my bad.